new-member-welcome.md

```diff
@@ -1,26 +1,17 @@
 # Welcoming new members
 
-Thanks a lot for joining the group. We’re still at the very early stages.
-
-Our main goals are around finding ways to improve email client HTML rendering with a strong focus on security, accessibility, resilience, performance, and future proofing.  Within the group we’re hoping to set up a number of projects to approach this from different angles. 
-
-We don’t have a leader or any hierarchy at the moment. As the group grows we may need to have a little more structure. 
-
-Remember to be respectful of all members of the group and all opinions. Please see the [Code of Conduct](code-of-conduct.md).
+Thank you for joining the EMC. We hope you can help us in [our mission](mission.md). Remember to be respectful of all members of the group and all opinions. Please see the [Code of Conduct](code-of-conduct.md).
 
 ## How we collaborate
-Currently we are discussing things in the [Email Standards Slack Channel](https://emailgeeks.slack.com/archives/C02C6TXA610) and on a weekly call on Tuesday at 5PM UK time (12 noon East Coast US / 9am West Coast US). [This link can help you convert to other timezones](https://www.inyourowntime.zone/2021-08-24_17.00_Europe.London?forget=1). (Be aware that clocks change in the UK on 31st October so the meeting time also changes then.)
-
-Once we have things more set up we are planning on cutting the weekly call back to a fortnightly or monthly call. We are also planning on a public forum to have more open discussion and allow more members of the community to share ideas. And a public website to publish our documentation.
+### Forum
+We aim to keep our discussions public whenever possible. This typically happens in the [Discussions](https://github.com/email-markup-consortium/email-markup-consortium/discussions) section of this repository which acts as our public forum.
 
-[Notes from previous calls can be found here](https://github.com/email-standards/email-standards/tree/main/meeting-notes). Also there is an agenda at the top of the page for the next call. Please feel free to add items for discussion.
+### Chat
+We also chat on Slack in the [Email Geeks Slack community](https://email.geeks.chat/) in the #email-markup-consortium channel. If you are an approved member and still do not have access to this channel, please contact [EMC admins](admins@emailmarkup.org).
```

Comment:
I think this needs to be a `mailto:` link

Reply:
Updated